_stbt/transition.py

```diff
@@ -22,11 +22,12 @@
 from .imgutils import FrameT
 from .logging import ddebug, debug, draw_on
 from .mask import MaskTypes
-from .types import KeyT, Region, SizeT
+from .types import KeyT, Region
 
 if typing.TYPE_CHECKING:
     # pylint:disable=unused-import
     from . import core
+    from .types import SizeT
```

Comment:
We already use `from __future__ import annotations` (see 905e2a98) and we already import `.types` so we don't need all this ceremony. I have pushed a commit.